res/database/Default/user/scr0llbaer.xml

```diff
@@ -453,18 +453,66 @@ A must for every housewife, a curse for hard men.</en>
+			<potential min="40" max="70" />
+			<outcome min="40" max="70" />
+			<review min="40" max="70" />
+			<speed min="40" max="70" />
 			<production_time value="60" />
 			<programme_data_modifiers>
 				<modifier name="topicality::age" value="5" />
 				<modifier name="topicality::refresh" value="2" />
 			</programme_data_modifiers>
 		</scripttemplate>
 
+		<scripttemplate product="3" licence_type="1" guid="40a79c72-f53d-4b4f-ad29-886dd6c99135" creator="9551" comment="TODO: Added after proposals from ronny and nittka, but perhaps this format should be more distinct from 'dynamite' somehow.">
+			<title>
+				<de>%pulse% - %show% %withexplosivenews%</de>
+				<en>%pulse% - The %show% %withexplosivenews%</en>
+			</title>
+			<description>
+				<de>Und noch ein Showbiz-Format. Die Welt da draußen scheint nicht genug davon bekommen zu können.</de>
+				<en>And another showbiz format. The world out there can't seem to get enough of it.</en>
+			</description>
+
+			<variables>
+				<pulse>
+					<de>Puls|Sprengstoff|Blitze|Buzz|Feuerwerk|Blitzlicht|Inferno|Zündstoff|Bumms|Furore</de>
+					<en>Pulse|Explosive|Flash|Buzz|Fireworks|Spotlight|Inferno|Fuse|Boom|Furore</en>
+				</pulse>
+				<show>
+					<de>Die Show|Der Ticker|Die Sendung|Das Magazin</de>
+					<en>Show|Ticker|Broadcast|Magazine</en>
+				</show>
+				<withexplosivenews>
+					<de>mit brisanten News|für Promi-News|mit aktuellen Skandalen|mit dem neuesten Klatsch und Tratsch|mit den heißesten Gerüchten|mit den neuesten Promi-Storys|mit den explosivsten Enthüllungen|mit den heißesten Gerüchten aus der Promi-Welt|mit den Knallern aus der Promi-Szene|mit den neuesten Skandalen und Enthüllungen</de>
+					<en>With Explosive News|for Celebrity News|With the Current Scandals|With the Latest Gossip and Rumors|With the Hottest Rumors|With the Latest Celebrity Stories|With the Most Explosive Revelations|With the Hottest Rumors From the Celebrity World|With the Blasts from the Celebrity Scene|With the Latest Scandals and Revelations</en>
+				</withexplosivenews>
+			</variables>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="4" required="1" />
+				<job index="2" function="8" required="1" />
+				<job index="3" function="128" required="1" />
+				<job index="4" function="64" required="0" />
+			</jobs>
+			<data flags="17" production_limit="100" production_broadcast_limit="2" production_broadcast_flags="512" production_licence_flags="36" />
+			<genres maingenre="301" subgenres="204,7" />
+			<blocks value="1" />
+			<price min="50000" max="100000" />
+			<potential min="20" max="50" />
+			<outcome min="20" max="50" />
+			<review min="20" max="50" />
+			<speed min="20" max="50" />
+			<production_time value="50" />
+			<programme_data_modifiers>
+				<modifier name="topicality::age" value="5" />
+				<modifier name="topicality::refresh" value="2" />
+			</programme_data_modifiers>
+			<availability script="TIME_YEARSPLAYED>1" />
```

Comment:
Just wondering if the "worse" format should be available later than the better one, but in a way it also makes sense - the origina vs. cheap copies.